src/e2e/specs/landing.spec.js

```diff
@@ -40,4 +42,83 @@ test.describe('Landing Page element verification', () => {
       timeout: 2000
     })
   })
+
+  test('Verify that the Landing Page content is displayed correctly', async ({ landingPage }) => {
+    // link to testrail case
+    test.info().annotations.push({
+      type: "testrail",
+      description: "https://testrail.stage.mozaws.net/index.php?/cases/view/2095093",
+    });
+
+    const links = await landingPage.heroSectionTexts()
+
+    // 1. Verify the "Find out if your personal information has been compromised" section.
+    await expect(async () => {
+      // verify hero title;
+      expect(await landingPage.heroHeader).toBe(links.heroHeaderText)
+
+      // verify hero description;
+      expect(await landingPage.heroDescription).toBe(links.heroDescriptionText)
+
+      // verify email field placeholder
+      expect(await landingPage.heroEmailInput.getAttribute('placeholder')).toBe(links.heroEmailInputPlaceholder)
+
+      // verify "Check for breaches" blue button;
+      await expect(await landingPage.heroCTAButton).toHaveScreenshot(
+        `${process.env.E2E_TEST_ENV}-heroCTAButton.png`
```

Comment:
How are these pngs generated?

Reply:
Yeah, as I mentioned in https://github.com/mozilla/blurts-server/pull/3332#discussion_r1308847119, if we generate these locally, they're bound to break (in fact, they wouldn't even match in the current state, as the CI tests don't run on `darwin`), so I'd expect these to be generated in CI, and there to be docs somewhere describing how to regenerate them (i.e. push a failing test run, and then download the artefacts with the mismatched screenshots).

Reply:
+1 , its already on the TODO list figure there's no rush especially when a big visual change is coming